scrubble-client/webpack.config.js

```diff
@@ -58,5 +60,11 @@ module.exports = {
           chunks: [pageName],
         })
     ),
+    new CleanWebpackPlugin(),
```

Comment:
idk about all this, i dont reallly understand it, just copy-and-paste from that medium article, which im sure is why this didnt work fix it 💯

Reply:
Looks like this deletes all the files from the output folder? Might be a good way to ensure a clean build every time, but also might be problematic to have this halfway through our plugins list